tests/foreman/api/test_syncplan.py

```diff
@@ -193,11 +193,21 @@ def test_positive_create_with_interval(self):
         :CaseImportance: Critical
         """
         for interval in valid_sync_interval():
-            with self.subTest(interval):
+            if interval != SYNC_INTERVAL['custom']:
                 sync_plan = entities.SyncPlan(
-                    interval=interval,
+                     description=gen_string('alpha'),
+                     organization=self.org,
+                     interval=interval
+                ).create()
+
+            elif (interval == 'custom cron'):
```

Comment:
simply else, as if iterval != SYNC_INTERVAL['custom'] the else case automatically should be equal
any way prefere to invert the condition: 
eg:
```python
if interval == SYNC_INTERVAL['custom']:
    ...
else:
    ...

```
what do you think?

Reply:
I agree that the second condition is redundant

Reply:
Thanks for the review. I will invert the condition.